src/ContainerFactory.ts

```diff
@@ -3,12 +3,8 @@ import { RedisClient, createClient } from 'redis';
 import { buildProviderModule } from 'inversify-binding-decorators';
 
 const createRedisClient = () => {
-    return createClient(
-        {
-            host: process.env.REDIS_HOST,
-            port: Number(process.env.REDIS_HOST)
-        }
-    ).on('error', (err) => { throw err; });
+    const redisUrl = `redis://${process.env.CACHE_SERVER}`;
```

Comment:
Ideally if you checked whether `process.env.CACHE_SERVER` is set and throw readable error when not set.